src/routes/(console)/project-[region]-[project]/sites/(components)/siteCard.svelte

```diff
@@ -59,13 +76,29 @@
 <Card padding="s" radius="m" {variant}>
     <Layout.Stack gap="l">
         <div class="card-grid">
-            <Image
-                border
-                radius="s"
-                ratio="16/9"
-                style="width: 100%; align-self: start"
-                src={getScreenshot($app.themeInUse, deployment)}
-                alt="Screenshot" />
+            {#if primaryDomain}
+                <a
+                    href={`${$regionalProtocol}${primaryDomain}`}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    aria-label="Open site">
```

Comment:
we have a `Card.Link` from Pink and a helper `Card` component with `href` prop in `src/lib/components` that we should use.